components/nav/NavBottom.vue

```diff
@@ -43,14 +43,14 @@ const { notifications } = useNotifications()
       </NuxtLink>
     </template>
     <NavBottomMoreMenu v-slot="{ toggleVisible, show }" v-model="moreMenuVisible" flex flex-row items-center place-content-center h-full flex-1 cursor-pointer>
-      <label
+      <button
         flex items-center place-content-center h-full flex-1 class="select-none"
         :class="show ? '!text-primary' : ''"
+        aria-label="More menu"
+        @click="toggleVisible"
       >
-        <input type="checkbox" z="-1" absolute inset-0 opacity-0 @click="toggleVisible">
-        <span v-show="show" i-ri:close-fill />
-        <span v-show="!show" i-ri:more-fill />
-      </label>
+        <span :class="show ? 'i-ri:close-fill' : 'i-ri:more-fill' />"
```

Comment:
`<span :class="show ? 'i-ri:close-fill' : 'i-ri:more-fill'" />`

The trailing `"`  is outside the span tag

Reply:
My snippet was wrong, sorry man

Reply:
Ah, that's why.😄 No problem, it was helpful and should be fixed now.